### ISSUE TITLE
protofmt - does not format option correctly

### ISSUE THREAD

**Issue by rogchap** [emicklei/proto-contrib#8]
Given the following:
```
rpc Hello (google.protobuf.Empty) returns (google.protobuf.Empty) {
  option (google.api.http) = {
    get: "/hello"
    additional_bindings: {
      get: "/hello/world"
    }
  };
}
```

it gets formatted as:
```
rpc Hello (google.protobuf.Empty) returns (google.protobuf.Empty) {
  option (google.api.http) = {
    get: "/hello"
    additional_bindings.get: "/hello/world"
  };

}
```

does not seem the work with the `additional_bindings` section

**Comment by emicklei** [comment]
thank you for reporting this.

**Comment by emicklei** [comment]
working on this issue

**Comment by emicklei** [comment]
checkout the PR (8issue) which fixes this.

**Comment by kalbasit** [comment]
@emicklei has this been closed in 55ea96e?
